packages/server/src/requestPipeline.ts

```diff
@@ -412,6 +413,24 @@ export async function processGraphQLRequest<TContext extends BaseContext>(
         requestContext as GraphQLRequestContextExecutionDidStart<TContext>,
       );
 
+      // If we don't have an operation, there's no reason to go further. We know
+      // `result` will consist of one error (returned by `graphql-js`'s
+      // `buildExecutionContext`).
+      if (!requestContext.operation && result.errors?.length) {
+        const error = result.errors[0];
+        const err = new OperationResolutionError(error.message, {
+          nodes: error.nodes,
+          originalError: error.originalError,
+          extensions: error.extensions,
+        });
+
+        await Promise.all(
```

Comment:
maybe throw the error instead of repeating this (so we never call executionDidEnd twice with errors).